docs/docs/guides/labs/components/integrations/sling-component-tutorial.md

```diff
@@ -0,0 +1,75 @@
+---
+title: 'Dagster & Sling with components'
+description: The dagster-sling library provides a SlingReplicationCollectionComponent, which can be used to represent a collection of Sling replications as assets in Dagster.
+sidebar_position: 200
+---
+
+import DgComponentsPreview from '@site/docs/partials/_DgComponentsPreview.md';
+
+<DgComponentsPreview />
+
+The [dagster-sling](/integrations/libraries/sling) library provides a `SlingReplicationCollectionComponent` which can be used to easily represent a collection of [Sling](https://slingdata.io/) replications as assets in Dagster.
+
+## 1. Prepare a Dagster project
+
+To begin, you'll need a Dagster project. You can use an [existing components-ready project](/guides/labs/dg/incrementally-adopting-dg/migrating-project) or create a new one:
+
+<CliInvocationExample path="docs_snippets/docs_snippets/guides/components/integrations/sling-component/1-scaffold-project.txt" />
+
+Activate the project virtual environment:
+
+```
```

Comment:
slightly visually weird that this doesn't have the `$` in front of the command like the other cli examples.